### ISSUE TITLE
TypeError: target.className.indexOf is not a function at Object.disableTocAnimation

### ISSUE THREAD

**Issue by romamilkovic** [tscanlin/tocbot#33]
Hi there, 

Thanks for amazing plugin! But I faced with bug using it. 

If you place some svg icon on the page and try to click it you'll see the error in your console:
```
Uncaught TypeError: target.className.indexOf is not a function
    at Object.disableTocAnimation (tocbot.js:549)
    at HTMLDocument.<anonymous> (tocbot.js:215)
    at HTMLDocument.<anonymous> (tocbot.js:128)
```

<img width="1437" alt="screen shot 2017-03-01 at 15 49 27" src="https://cloud.githubusercontent.com/assets/10784316/23450538/b35a8f94-fe96-11e6-963d-810097e51cee.png">

**Comment by tscanlin** [comment]
Hey, thanks for reporting this bug. And glad to hear you like Tocbot. :)

I was just working on Tocbot earlier and already have a fix for this bug. Turns out classname isn't a string for SVGs.

**Comment by tscanlin** [comment]
This is now fixed in `2.3.1`. Thanks again!
